rotkehlchen/inquirer.py

```diff
@@ -322,7 +350,10 @@ def _query_oracle_instances(
         )
         price = Price(ZERO)
         for oracle, oracle_instance in zip(oracles, oracle_instances):
-            if oracle_instance.rate_limited_in_last() is True:
+            if (
+                isinstance(oracle_instance, PriceOracleInterface) and
```

Comment:
Is this the historical one? `_query_oracle_instances` is only used for current price

Reply:
yes. I used `PrIceOracleInterface` for oracles that can query both historical and current prce and then `CUrrentPriceOracle` for the ones that only query the current.